ssl/s3_enc.c

```diff
@@ -537,7 +537,7 @@ int ssl3_digest_cached_records(SSL *s, int keep)
         }
 
         /* Loop through bits of algorithm2 field and create MD_CTX-es */
-        for (i = 0; ssl_get_handshake_digest(i, &mask, &md); i++) {
+        for (i = 0; ssl_get_certverify_digest(i, &mask, &md); i++) {
```

Comment:
I want to ensure that any digest necessary for cert_verify is calculated. I was not sure how to implement it so I provided a separate function for enumerating them.